integ-test/build.gradle

```diff
@@ -91,7 +91,7 @@ dependencies {
     testRuntimeOnly('org.junit.jupiter:junit-jupiter-engine:5.6.2')
 
     testImplementation group: 'com.h2database', name: 'h2', version: '2.1.214'
-    testImplementation group: 'org.xerial', name: 'sqlite-jdbc', version: '3.28.0'
+    testImplementation group: 'org.xerial', name: 'sqlite-jdbc', version: '3.32.3.3'
```

Comment:
I had to update the version of sqlite because the older version didn't work on M1 Macs

Reply:
This change should be in upstream/main.  If you downmerge into your branch(es) you should get this fix for free. 

```
git pull upstream
git checkout integ-date-time-correctness
git merge upstream/main
git checkout dev-date-time-correctness
git merge integ-date-time-correctness
```

Reply:
I have pulled the new changes